website/docs/sql-function-glossary/sql-aggregate-functions/sql-avg.md

```diff
@@ -0,0 +1,51 @@
+---
+id: sql-avg
+title: SQL AVG
+description: The AVG function is used to straightforwardly calculate the average of a numeric column, but you may also see it used in a window function to calculate rolling averages.
+slug: /sql-function-glossary/sql-avg
+---
+
+<head>
+    <title>Working with the SQL AVG function</title>
+</head>
+
+You’re a data person, so we assume you’re going to be calculating averages of some metrics \**waves hands airly*\* at some point in your career. And the way to calculate averages of a numeric column in SQL is by using the AVG function.
+
+## How to use the AVG function
+
+The AVG function is a part of the group of mathematical or aggregate functions (ex. MIN, MAX, SUM) that are often used in SQL to summarize datasets. You’ll most likely see the AVG function used to straightforwardly calculate the average of a numeric column, but you may also see it used in a window function to calculate rolling averages.
+
+### AVG function example
+
+```sql
+select
+	date_trunc('month, order_date) as order_month,
+	round(avg(amount)) as avg_order_amount
+from {{ ref('orders') }}
+where status not in ('returned', 'return_pending')
+group by 1
+```
+
+This query using the [Jaffle Shop’s](https://github.com/dbt-labs/jaffle_shop) `orders` table will return the rounded order amount per each order month:
```

Comment:
May want to intro Jaffle Shop as "the dbt sample dataset, Jaffle Shop" as organic searchers won't know it